kyuubi-server/src/main/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilder.scala

```diff
@@ -54,27 +54,22 @@ class FlinkProcessBuilder(
     val buffer = new ArrayBuffer[String]()
     buffer += s"bash"
     buffer += s"-c"
-    val commandStr = new StringBuilder()
+    val commandSet = new LinkedHashSet[String]()
```

Comment:
why not use the `buffer`

Reply:
I need to treat the contents contained in the commandSet as a whole string, while separating the contents with spaces. this string as an argument to bash -C

Reply:
i don't quite get it